src/main/java/com/tg5/controller/ReportsController.java

```diff
@@ -0,0 +1,30 @@
+package com.tg5.controller;
+
+
+import com.tg5.service.reports.AttendanceDownloadService;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RestController
+@RequestMapping("/reports")
+public class ReportsController {
```

Comment:
We shouldn't need any extra controller, just a single reports and attendance, that returns json/excel based on accept header.